samples/graph-app-installation-lifecycle/csharp/README.md

```diff
@@ -18,6 +18,8 @@ This sample app demonstarte the installation lifecycle for Teams [Apps](https://
 
 - Register your app with Microsoft identity platform via the Azure AD portal
   - Your app must be registered in the Azure AD portal to integrate with the Microsoft identity platform and call Microsoft Graph APIs. See [Register an application with the Microsoft identity platform](https://docs.microsoft.com/en-us/graph/auth-register-app-v2). 
+- You need to add following permissions mentioned in the below screenshots to call respective Graph   API
+![](https://user-images.githubusercontent.com/50989436/115846799-e2ce5500-a43f-11eb-984a-b16aec0bc586.png))
```

Comment:
These are not required. Please check and update.

Reply:
updated